partials/Channels.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
```

Comment:
It would be great not to have to import React.

Reply:
I will try to do that with newest preset for React, but I don't know if this will work with our case, but should :)

Reply:
Fixed 🚀

Reply:
👏